apps/dashboard/src/components/stack-companion/unified-docs-widget.tsx

```
@@ -0,0 +1,337 @@
+  const handleGoBack = () => {
+    try {
+      if (iframeRef?.contentWindow) {
+        // Try to go back in iframe history
+        iframeRef.contentWindow.history.back();
+      }
+    } catch (error) {
+      // If we can't access iframe history, try to reload the previous page
+      // This is a fallback that at least resets the iframe
+      console.warn('Cannot access iframe history, reloading current page');
+      if (iframeRef) {
+        iframeRef.src = iframeRef.src;
```

Biome (2.1.2)

[error] src is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)